packages/cloud-cognitive/src/components/WebTerminal/WebTerminal.stories.js

```diff
@@ -51,13 +53,30 @@ const Template = (args) => {
 };
 
 export const Default = prepareStory(Template, {
-  args: {},
+  args: { documentationLinks },
 });
 
 export const WithDocumentationLinks = prepareStory(Template, {
   args: { documentationLinks },
 });
 
+export const WithActions = prepareStory(Template, {
+  args: {
+    actions: [
+      {
+        renderIcon: Code,
+        onClick: () => alert('clicked on action'),
```

Comment:
For consistency, we should use storybook action here

```jsx
import { action } from '@storybook/addon-actions';
...
...
actions: [
      {
        renderIcon: Code,
        onClick: action('Clicked action')
       }
],
...
...

```

Reply:
Weird, I didn't realize you don't pass it as a callback. Thanks for pointing this out